### ISSUE TITLE
IME input method on iced application (fcitix on linux)

### ISSUE THREAD

**Issue by Piping** [iced-rs/iced#979]
Hi,

is there a plan to support input method on linux for iced application?

The current state of iced example looks like this:

 <img src="https://user-images.githubusercontent.com/12042284/127789818-a1e4280b-f80f-48f9-b9a6-d503c3e8d678.png" width="300" height="300">

The input method (fcitix)'s panel does not follow the cursor.

Possible way to do is via XIM in linux?

**Comment by 13r0ck** [comment]
This is being worked on in #686

**Comment by hecrj** [comment]
#33 is also blocking this, since we cannot draw text properly yet!

**Comment by lemon-rs** [comment]
winit added IME support in 0.27.0.

**Comment by ksqsf** [comment]
IME on macOS does not work either.

**Comment by t18b219k** [comment]
we need to change input model to fit winit IME event.
almost same as ImeEvent api in web.

**Comment by t18b219k** [comment]
Implemented IME for text_input . we need more work for platform native performance.

https://user-images.githubusercontent.com/48007646/196032903-61fa6825-aa20-449f-95cc-f8db4044b066.mp4

**Comment by hecrj** [comment]
Basic IME support has landed on `master` thanks to #2777. Check it out!

**Comment by dcampbell24** [comment]
How does IME support work (on master)? When I type in this window with Mozc Hiragana enabled I get a box with what Hiragana I want to input appearing, but with iced I get nothing.
